tests/integration/test_db_migration.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""Discourse integration tests."""
+
+import logging
+
+import pytest
+from botocore.config import Config
+from ops.model import WaitingStatus
+from pytest_operator.plugin import Model, OpsTest
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_db_migration(model: Model, ops_test: OpsTest, pytestconfig: Config, run_action):
+    """
+    arrange: preload postgres with a mock db that is created in Discource v3.2.0
+    act: deploy and integrate with discourse v3.3.0
+    assert: discourse must be active idle, previously it was creating migration
+    errors related to not being able to delete some columns because of triggers
+    """
+    postgres_app = await model.deploy(
+        "postgresql-k8s",
+        channel="14/stable",
+        series="jammy",
+        revision=300,
```

Comment:
Do we really need to specify a particular revision here?

Reply:
Not really but it was specified in https://github.com/canonical/discourse-k8s-operator/blob/50e229a3ef8e5ac0f8dbed2398a276c7833d2e6b/tests/integration/conftest.py#L155

Reply:
Ok, I agree consistency is better. Will close out this comment thread.